### PR TITLE
Release notes for 0.8.3

### PR DESCRIPTION
**Pull request by johannes-b**
Signed-off-by: Johannes <johannes.braeuer@dynatrace.com>

### CODE REVIEW

File: releasenotes/releasenotes_develop.md [hunk 1]
```diff
@@ -1,42 +1,141 @@
...
+# Release Notes 0.8.3
 
-Keptn 0.x.x is
+Keptn 0.8.3 implements the Keptn Enhancement Proposal [#37](https://github.com/keptn/enhancement-proposals/pull/37) for allowing a Keptn user to customize the remediation sequence in the Shipyard. Besides, improvements for the user experience in the Bridge are implemented like setting the Git upstream repository or linking various screens. 
 
 ---
 
 **Key announcements:**
 
+:tada: *Customization of auto-remediation sequences*: With this release, it is possible to customize the remediation sequences, which take care of resolving an open problem for a service. Therefore, the remediation sequence can be modeled in the Shipyard for a specific stage. Besides, it is possible to let the *action provides* run on an execution plane.
```
Comment (bacherfl) [inline releasenotes/releasenotes_develop.md:9]: typo: action-provides -> action provider

File: releasenotes/releasenotes_develop.md [hunk 2]
```diff
@@ -1,42 +1,141 @@
...
+    tasks:
+    - name: get-action 
+    - name: action
+    - name: evaluation
+      triggeredAfter: "10m"
+      properties:
+        timeframe: "10m"
+  ```
+
+:star: *Length of service name increased to 49 characters*: The limitation of the service name length has been loosened to allow a length of 43 characters.
```
Comment (bacherfl) [inline releasenotes/releasenotes_develop.md:29]: 49 or 43?

Comment (bacherfl) [reply releasenotes/releasenotes_develop.md:29]: Just checked - it's 43 :)

Comment (johannes-b) [reply releasenotes/releasenotes_develop.md]: fixed it ;)

File: releasenotes/releasenotes_develop.md (hunk 1 above)
Comment (johannes-b) [reply releasenotes/releasenotes_develop.md]: thx

**Review by bacherfl — APPROVED** [review]

**Pull request merged**